sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -2141,7 +2141,9 @@ abstract class DDLSuite extends QueryTest with DDLSuiteBase {
       root = Utils.createTempDir().getCanonicalPath, namePrefix = "addDirectory")
     val testFile = File.createTempFile("testFile", "1", directoryToAdd)
     spark.sql(s"ADD FILE $directoryToAdd")
-    assert(new File(SparkFiles.get(s"${directoryToAdd.getName}/${testFile.getName}")).exists())
+    spark.artifactManager.withResources {
```

Comment:
Is add file isolated? If not, then why do we need this?

Reply:
Now it's hacky: The JAR will be added to SparkContect with a session UUID prefix but will be accessible by all sessions (due to we wrapped it's classloader).

On a second thought, I could make it cleaner by passing `default` as UUID into the `sparkContext.addJar` method to explicitly disable isolation.

Reply:
This is a leftover from another PR that enables Hive isolation. Removed!

Reply:
I found that my previous comment was wrong. `ADD JAR` is isolated when used outside of Hive.
To not bring a behaviour change, I'll make a small change to disable isolation for this command even outside Hive.

Reply:
And we can discuss whether we should re-enable isolation as a follow-up.